auditbeat/auditbeat.reference.yml

```diff
@@ -1133,6 +1133,9 @@ setup.kibana:
   # Optional HTTP path
   #path: ""
 
+  # Optional Space ID.
```

Comment:
```suggestion
  # Optional Kibana space ID.
```

Reply:
I think it's worth qualifying here just like you do in the ref docs. Make this change globally.